miniml/compiler/compile.scm

```diff
@@ -905,7 +905,38 @@
 (define BNEQ 132)
 (define STOP 143)
 (define RERAISE 146)
-(define obj_tag (prim "caml_obj_tag"))
+(define obj_tag (cdr (prim "caml_obj_tag")))
+
+; bindings store for each name a definition, but also some visibility
+; information, which indicate if this name was defined as "exported"
+; by the current compilation unit ('Export), or is just available locally ('Local).
+; In OCaml, this is the definition between `include M` and `open M`.
```

Comment:
[typo] shouldn't this be "difference" instead of "definition"?

Reply:
Indeed, thanks! I fixed this and rebased the PR on top of the new master.